stable/k8s-spot-termination-handler/README.md

```diff
@@ -42,3 +42,4 @@ Parameter | Description | Default
 `nodeSelector` | node labels for pod assignment | `{}`
 `tolerations` | node taints to tolerate (requires Kubernetes >=1.6) | `[]`
 `affinity` | node/pod affinities (requires Kubernetes >=1.6) | `{}`
+`priorityClassName` | pod priorityClassName for pod. | ``
```

Comment:
default seems to be `system-node-critical`.
Either set it to "" as documented or update docs if `system-node-critical` should in fact be default

Reply:
Changed the value to "" since the default should be empty